requestor/service_manager/yapapi_connector.py

```diff
@@ -0,0 +1,53 @@
+from yapapi.executor import Golem
+import asyncio
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from .service_wrapper import ServiceWrapper
+
+
+class YapapiConnector():
+    def __init__(self, executor_cfg: dict):
+        self.executor_cfg = executor_cfg
+
+        self.command_queue = asyncio.Queue()
+        self.run_service_tasks = []
+        self.executor_task = None
+
+    def create_instance(self, service_wrapper: 'ServiceWrapper'):
+        if self.executor_task is None:
+            self.executor_task = asyncio.create_task(self.run())
```

Comment:
Couldn't this be done in `__init__`?

Reply:
It could be in `__init__`, but than we would need a running loop when `yapapi_connector` is initialized.
E.g. if you change it that way, `run_service_manager.py` will no longer work, because initialization is outside the `async` code.
--> this way there is more flexibility from the end user
--> imho this is better
but if you think otherwise, I could change this.